src/hooks/gatewayAuth/cache.ts

```diff
@@ -0,0 +1,59 @@
+import { isBefore } from 'date-fns';
+import { decodeJwt, JWTPayload } from 'jose';
+import { LRUMap } from 'mnemonist';
+import { TaskAuthorizationResponse } from 'utils/dataPlane-utils';
+
+// The limit of 10 is totally arbitrary. Felt like an okay number so all 3 tables could have
+//  something in cache and allow someone some flexibility to search
```

Comment:
We should rewrite this comment now. I think just calling out it is generally arbitrary is enough.